src/config.cpp

```diff
@@ -37,6 +37,7 @@ bool load_configuration(const std::string& path, config_type& configuration){
 
         if (file.is_open() && file.good()) {
             std::string line;
+            std::cout << "INFO: Reading config file " << path << std::endl;
```

Comment:
Unfortunately, we have to remove this output line. If we use the tool in command line mode, this line will appear every single time. I only output stuff when there is an error or when the server is running. And before the configuration is loaded, we don't know if the server is running.

Reply:
Ah, that makes sense. Removed.